src/imgtools/cli/dicomsort.py

```
@@ -1,5 +1,5 @@
 import pathlib
-
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)